x-pack/plugins/canvas/public/components/page_manager/page_manager.js

```diff
@@ -185,13 +197,24 @@ export class PageManager extends React.PureComponent {
         </EuiFlexGroup>
         <ConfirmModal
           isOpen={deleteId != null}
-          title="Remove Page"
-          message="Are you sure you want to remove this page?"
-          confirmButtonText="Remove"
+          title={this.props.intl.formatMessage({
```

Comment:
Please allocate `this.props.intl` into `intl` constant (line 154) and use it here and bellow:
```
const { pages, addPage, deleteId, intl } = this.props;
...
title={intl.formatMessage({
....
```